minio/parsers.py

```diff
@@ -348,6 +358,7 @@ def parse_location_constraint(data):
     root = S3Element.fromstring('BucketLocationConstraintResult', data)
     return root.text()
 
+
 def _iso8601_to_localized_time(date_string):
     """
     Convert iso8601 date string into UTC time.
```

Comment:
I don't think this docstring is accurate as it returns a localized time, hence the refactoring to extract `_iso8601_to_utc_datetime`